src/utils.js

```diff
@@ -127,25 +127,62 @@ exports.hideAbsolutePathsInObject = (object) => {
   }
 
   if (_.isString(object)) {
-    const words = object.split(' ').map(word => (path.isAbsolute(word) ? path.basename(word) : word))
-    return words.join(' ')
+    const words = object.split(' ')
```

Comment:
We probably want to split spaces or newlines here.

Reply:
And we split it with spaces, what's your doubt here?